frontend/sentiment-dashboard/src/app/app-routing.module.ts

```diff
@@ -15,7 +17,7 @@ const routes: Routes = [
   {
     path: 'settings',
     component: SettingsComponent,
-    canActivate: [AuthGuardService]
+    canActivate: [AuthGuardService, AdminGuard]
```

Comment:
Why do we need both AuthGuardService and AdminGuard here? What's the difference between the two? Not challenging anything, just trying to understand more clearly since I've never used canActivate before :)

Reply:
AuthGuard restricts the page to logged in users. AdminGuard restricts the page to users who are in the admin group. Also I should probably rename AuthGuardService just to be consistent